lib/service_check.pm

```diff
@@ -51,79 +53,92 @@ our $default_services = {
     postfix => {
         srv_pkg_name  => 'postfix',
         srv_proc_name => 'postfix',
-        support_ver   => '12-SP3,12-SP4,12-SP5,15,15-SP1'
-    },
-    # Quick hack for poo 50576, we need this workround before full solution
-    #    apache => {
-    #    srv_pkg_name  => 'apache2',
-    #    srv_proc_name => 'apache2',
-    #    support_ver   => '12-SP3,12-SP4,15,15-SP1'
-    #},
+        support_ver   => '12-SP3,12-SP4,15,15-SP1'
```

Comment:
how remove of SP5 for postfix related to "extending of apache testing" ?

Reply:
changed, cause weigao submitted a PR and I'm not update the file.